source/inference.md

```diff
@@ -1043,7 +1037,7 @@ alt.Chart(six_bootstrap_samples, height=150).mark_bar().encode(
         .title("Price per night (dollars)"),
     y=alt.Y("count()").title("Count")
 ).facet(
-    "replicate",
+    "replicate:N",  # Remeber that `:N` converts the variable to a categorical type
```

Comment:
These two occurrences of a suffix inside `facet` are difference than the 5.1 regression. Details are in https://github.com/altair-viz/altair/issues/3138, but in brief altair suppresses some warnings from vega-lite while vegafusion does not. So after switching the data_transformer we would see the following warning without the suffix:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/413778b5-7827-41e3-be56-fc2194297110)